src/store/actions/createComment.js

```diff
@@ -0,0 +1,31 @@
+import { CREATE_COMMENT } from 'store/constants'
+import { uniqueId } from 'lodash/fp'
+import { AnalyticsEvents } from 'hylo-utils/constants'
+import CreateCommentMutation from 'graphql/mutations/CreateCommentMutation.graphql'
+
+export default function createComment ({
+  postId,
+  text,
+  attachments
+}) {
+  return {
+    type: CREATE_COMMENT,
+    graphql: {
+      query: CreateCommentMutation,
+      variables: {
+        postId,
+        text,
+        attachments
+      }
+    },
+    meta: {
+      optimistic: true,
+      extractModel: 'Comment',
+      tempId: uniqueId(`post${postId}_`),
```

Comment:
what is this for?

Reply:
That predated my work and could speculate, but essentially I just left it as is.